test/unit/test_blas_tile/test_trsm.h

```diff
@@ -28,7 +28,7 @@ using namespace dlaf_test;
 using namespace testing;
 
 /// @brief Returns el_op_a, el_b, res_b for side = Left.
-template <class T>
+template <class T, class ElementIndex>
```

Comment:
I would put `ElementIndex` as first template parameter, since T can be deduced from the arguments.